### DISCUSSION TITLE
With the shuttering of GL, possibility to use Labmem with JP?

### DISCUSSION THREAD

**Discussion by Shiroimaken** [HughJeffner/FFRK-LabMem#230] [Ideas]
Basically what it says on the tin. I'd be happy to provide pictures and whatever other materials might be necessary for Labmem to function properly.

**Comment by HughJeffner** [comment]
I don't have a JP account and don't plan on migrating so I won't be doing it personally, but some others might.

Check out this fork:
https://github.com/bover87/HMB-FFRK-JP-LabMem

**Reply by Shiroimaken** [reply]
Thanks, I'll check it out.

**Comment by Vandalhart** [comment]
I'm actually running that JP branch, but I'm running into some timeouts on the "Moving on..." message in some scenarios. Should I try to contact @bover87 to see about resolving that? I'm still trying to determine in what scenarios it happens, but it feels like the confirm button doesn't load fast enough for the program to click it and I'm not sure what timing to update to maybe solve it. 

Also, I'm getting asked to reinstall the certificate every time I run Labmem. I have renamed the folder compared to it's original filename.

**Comment by HughJeffner** [comment]
Not sure about the time outs, and it looks like the forked repo didn't enable posting issues.  They did do an update a couple days ago though.

The certificate issue is usually due to multiple versions of the certificate installed.  If your device requires root then it can be a bit difficult to fix, otherwise you can clear all trusted credentials and re-do the cert install procedure.

**Reply by Vandalhart** [reply]
I did originally have folders for both your version, and this JP version in seperate locations. I have since removed the english folders but I'm still seeing the reinstall certificate message on labmem start up.

I usually do the "remove user credentials step", then reinstall the prompted one, and hit enter in labmem. It does work, but I just don't remember it doing it for your original branch.

Could I have some left over information from the original install somewhere?

**Reply by bover87** [reply]
First, make sure you're using 720x1280 window size for your emulator. Other sizes can work but the bot code is written assuming that resolution.

Second, try resetting your offsets. In the bot options, set both offsets to -1, then hit Alt + O at the title screen (the one with the core character image and the START button).

If none of that works, the only thing I can think of is that you might not have enough memory or CPU power to run the bot reliably. Neither the bot nor the emu is extremely demanding and any reasonably modern rig should be able to run them, but if you're running other tasks alongside them you can run into problems (on a basic PC not designed for gaming or other high-intensity uses, even having too many browser tabs open can cause performance problems with the emu, which in turn can cause the bot to register hangs).

Also @HughJeffner I've enabled issues on my fork.

**Comment by Vandalhart** [comment]
The window size is set to 720x1280 (360dpi). When I hit Alt-0 I get "Ignoring because Restart Lab control not enabled", if I enable it and do it, I get "Disabling lab after this run".

This PC isnt over new, about 5 years, but the only other thing I do while it's running is watch youtube videos. I notice it even does it if I'm just letting run while not using the PC.

If you want to move conversation to your own repo, I can do so.

Derp.. it's Alt-"O" not Alt-"0"

**Reply by bover87** [reply]
That's up to Hugh since this isn't my page.

Make sure you're using the right command for the offsets--it's Alt + O (the letter), not 0.

**Reply by Vandalhart** [reply]
The first instance after updating the offsets may have solve the problem. I'll keep an eye on it and follow up if there's any further problems.
Thank you for the replies.
